homeassistant/components/loxone.py

```diff
@@ -0,0 +1,255 @@
+"""
+Component to create an interface to the Loxone Miniserver
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/loxone/
+"""
+# pylint: disable=unused-import, too-many-lines
+import asyncio
+import codecs
+import hashlib
+import hmac
+import logging
+import uuid
+import json
+from datetime import timedelta
+from struct import unpack
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_HOST, CONF_PORT, CONF_USERNAME,
+                                 CONF_PASSWORD)
+from homeassistant.helpers.event import async_track_time_interval
+
+REQUIREMENTS = ['websockets==3.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'loxone'
+DEFAULT_HOST = '127.0.0.1'
```

Comment:
This doesn't make much sense. Or isn't the Miniserver an independent hardware unit only?